pylint/extensions/docparams.py

```diff
@@ -393,6 +404,33 @@ def _compare_different_args(
                 node=warning_node,
             )
 
+    def _compare_ignored_args(
+        self,
+        found_argument_names,
+        message_id,
+        ignored_argument_names,
+        warning_node,
+    ):
+        """Compare the found argument names with the ignored ones and
+        generate a message if there are ignored arguments found.
+
+        :param set found_argument_names: argument names found in the
+            docstring
```

Comment:
I am with you on that, it was new for me but it is the style that is being used on the file as you can see on the existing methods: https://github.com/PyCQA/pylint/pull/3842/files/2378dc143d99e66febb1c3daeb66fbbb1b3e7d7c#diff-61228d78a7c97a0961324b4e7e6eb65eR383

I then chose to keep the same style instead of mixing them.

Happy to change for what you think would be more appropriate. What is your opinion on that?

Reply:
@luigibertaco it seems as this style is used only in the `_compare_missing_args`, `_compare_ignored_args` and `_compare_different_args` methods. I suppose it is a mistake. Do you mind correcting it?

Reply:
Hi @hippo91, I've fixed the sphinx style on the comments, but it got me thinking if pylint shouldn't have raised a warning for that.